go-controller/pkg/ovn/egressip_test.go

```diff
@@ -11536,3 +11699,76 @@ func getSwitchManagementPortIP(node *v1.Node) (net.IP, error) {
 	}
 	return nil, fmt.Errorf("failed to find management port IP for node %s", node.Name)
 }
+
+/*
+func getEIPAddrSet(name egressIpAddrSetName, ip string) (*nbdb.AddressSet, *nbdb.AddressSet) {
+	v4, v6 := addressset.GetDbObjsForAS(getEgressIPAddrSetDbIDs(name, DefaultNetworkControllerName),
+		[]net.IP{net.ParseIP(ip)})
+	v4.UUID = v4.Name + "UUID"
+	v6.UUID = v6.Name + "UUID"
+	return v4, v6
+}
+*/
+
+// returns the address set with externalID "k8s.ovn.org/name": "egresssvc-served-pods"
+func getEgressSVCServedPodsAS(ips []string) (*nbdb.AddressSet, *nbdb.AddressSet) {
+	dbIDs := egresssvc.GetEgressServiceAddrSetDbIDs(DefaultNetworkControllerName)
+	egressSvcPodsV4, egressSvcPodsV6 := addressset.GetHashNamesForAS(dbIDs)
+	v4ips, v6ips := separateIPs(ips)
+	v4 := createAS(egressSvcPodsV4, egresssvc.EgressServiceServedPodsAddrSetName, "v4", string(ops.EgressServiceOwnerType), dbIDs, v4ips)
+	v6 := createAS(egressSvcPodsV6, egresssvc.EgressServiceServedPodsAddrSetName, "v6", string(ops.EgressServiceOwnerType), dbIDs, v6ips)
+
+	return v4, v6
+}
+
+// returns the address set with externalID "k8s.ovn.org/name": "egressip-served-pods""
+func getEgressIPServedPodsAS(ips []string) (*nbdb.AddressSet, *nbdb.AddressSet) {
+	dbIDs := getEgressIPAddrSetDbIDs(EgressIPServedPodsAddrSetName, DefaultNetworkControllerName)
+	egressIPServedPodsV4, egressIPServedPodsV6 := addressset.GetHashNamesForAS(dbIDs)
+	v4ips, v6ips := separateIPs(ips)
+	v4 := createAS(egressIPServedPodsV4, string(EgressIPServedPodsAddrSetName), "v4", string(ops.EgressIPOwnerType), dbIDs, v4ips)
```

Comment:
will https://github.com/ovn-org/ovn-kubernetes/blob/47fcdd2607be4e4b9cc198a0d6fca385bf4030ed/go-controller/pkg/ovn/address_set/address_set.go#L325 work instead of createAS everywhere?

Reply:
Good spot, I want to make this code easier to read by not rewritting test functions everywhere that do the same thing and here I am doing just that